doc/api/esm.md

```diff
@@ -0,0 +1,85 @@
+# Modules
+
+<!--introduced_in=v9.x.x-->
+
+> Stability: 1 - Experimental
+
+<!--name=esm-->
+
+Node contains support for ES Modules based upon [the Node EP for ES Modules](https://github.com/nodejs/node-eps/blob/master/002-es-modules.md).
+
+Not all features of the EP are complete and will be landing as both VM support and implementation is ready. Error messages are still being polished.
+
+## Enabling
+
+<!-- type=misc -->
+
+The `--experimental-modules` flag can be used to enable features for loading ESM modules.
+
+Once this has been set, you can use files ending with `.mjs` as ES Modules.
+
+```sh
+node --experimental-modules my-app.mjs
+```
+
+## Features
+
+<!-- type=misc -->
+
+### Supported
+
+Only the CLI argument for the main entry point to your program can be an entry point into an ESM graph. In the future you can use `import()` to create entry points into ESM graphs at run time.
```

Comment:
Please avoid the use of `you` and `your` in the docs. I know this is just an informal start to the docs but still worthwhile.

```
Currently, using the `--experimental-modules` command line argument and passing
an `*.mjs` file as the main application script to run is the only supported entry point
to using ESM. In the future, `import()` will be supported as a run-time entry point.
```